rules/aws/exfiltration_s3_data_management_tampering.toml

```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2021/05/05"
+maturity = "production"
+updated_date = "2021/05/05"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = "Identifies when threat actors tamper with S3 data management in Amazon Web Services"
```

Comment:
There is no guarantee that a threat actor performed this action. Refer to some of the other AWS rules, which have a good structure for their descriptions.